R/Features-constructors.R

```diff
@@ -88,7 +101,11 @@ readFeatures <- function(table, ecol, fnames, ..., name = NULL)  {
     }
     se <- SummarizedExperiment(assay,
```

Comment:
No need to create `se` to then return it right away. Just use the call to the constructor as last expression.